drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -172,12 +172,12 @@ void at86rf2xx_set_page(at86rf2xx_t *dev, uint8_t page)
 
 uint16_t at86rf2xx_get_pan(at86rf2xx_t *dev)
 {
-    return dev->pan;
+    return byteorder_ntohs(dev->pan);
 }
 
 void at86rf2xx_set_pan(at86rf2xx_t *dev, uint16_t pan)
 {
-    dev->pan = pan;
+    dev->pan = byteorder_htons(pan);
```

Comment:
Quote 802.15.4-2011 (emphasis mine):

> ### 5.2 MAC frame formats
This subclause specifies the format of the MAC frame (MPDU).
The frames in the MAC sublayer are described as a sequence of fields in a specific order. All frame formats in this subclause are depicted in the order in which they are transmitted by the PHY, from left to right, where the leftmost bit is transmitted first in time. **Bits within each field are numbered from 0 (leftmost and least significant) to k – 1 (rightmost and most significant)**, where the length of the field is k bits. **Fields that are longer than a single octet are sent to the PHY in the order from the octet containing the lowest numbered bits to the octet containing the highest numbered bits.**